ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/ImageLoader.qml

```diff
@@ -49,6 +51,39 @@ Item {
             }
         }
 
+        layer.enabled: true
+        layer.effect: OpacityMask {
+            maskSource: Item {
+                width: imageMessage.width
+                height: imageMessage.height
+
+                Rectangle {
+                    anchors.top: parent.top
+                    anchors.left: parent.left
+                    width: imageMessage.width
+                    height: imageMessage.height
+                    radius: 16
+                }
+
+                Rectangle {
+                    anchors.bottom: parent.bottom
+                    anchors.left: parent.left
+                    width: 32
+                    height: 32
+                    radius: 4
+                    visible: !imageContainer.isCurrentUser
+                }
+                Rectangle {
+                    anchors.bottom: parent.bottom
+                    anchors.right: parent.right
+                    width: 32
+                    height: 32
+                    radius: 4
+                    visible: imageContainer.isCurrentUser
+                }
```

Comment:
You only need one rectangle and change its position:
```suggestion
                Rectangle {
                    anchors.bottom: parent.bottom
                    anchors.right:  imageContainer.isCurrentUser ? parent.right : undefined
                    anchors.left:  imageContainer.isCurrentUser ? undefined : parent.left
                    width: 32
                    height: 32
                    radius: 4
                    visible: imageContainer.isCurrentUser
                }
```

Reply:
Tried this, didn't work. For some reason the Rectangle aren't drawn anymore at all when done this way. Showing/hiding two rectangles worked though.

Reply:
That's weird. Like none of them showed or they showed both at the same time?

Reply:
Non of them showed at all